camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex

```diff
@@ -1,27 +1,39 @@
 defmodule Membrane.Demo.CameraToHls.Pipeline do
   use Membrane.Pipeline
 
+  alias Membrane.HTTPAdaptiveStream.Manifest
+  alias Membrane.Time
+  import Membrane.ChildrenSpec
+
   @impl true
-  def handle_init(_) do
-    children = [
+  def handle_init(_ctx, _opts) do
+    segment_duration = %Manifest.SegmentDuration{min: Time.seconds(2), target: Time.seconds(5)}
```

Comment:
`SegmentDuration` was dropped in https://github.com/membraneframework/membrane_http_adaptive_stream_plugin/commit/1458d37fc43500b9c54992968433c27d51a3dc83 (v0.15.0).

Reply:
Indeed, good spot!
Unfortunately this PR has been abandoned so I am closing it in favour of #241 and #239, which update all the demos to Core v0.12 (including `camera_to_hls`).